### PR TITLE
[DASH-2198] Fix "not closed tag" error on templates view

### PR DESCRIPTION
**Pull request by wpasinski**
- I also added `-webkit-font-smoothing: antialiased` to get font more sharp on high dpi displays

### CODE REVIEW

File: src/common/Editor/Editor.jsx [hunk 1]
```diff
@@ -55,19 +34,22 @@ export default React.createClass({
   },
 
   componentDidMount() {
-    this.editor = ace.edit(this.props.name);
+    const { name, mode, theme, fontSize, onChange, value, showGutter, minLines, maxLines, readOnly, highlightActiveLine,
+      setShowPrintMargin, onLoad } = this.props;
```
Comment (Flixow) [inline src/common/Editor/Editor.jsx]: What about move every prop to new line?

File: src/apps/Template/Template.jsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ const Template = React.createClass({
   },
 
   componentWillUnmount() {
-    Store.clearTemplate();
+    TemplateStore.clearTemplate();
```
Comment (patrykkopycinski) [inline src/apps/Template/Template.jsx]: it should be done via actions

File: src/apps/Template/Template.jsx [hunk 3]
```diff
@@ -289,9 +289,26 @@ const Template = React.createClass({
     );
   },
 
+  renderCodeEditor() {
```
Comment (patrykkopycinski) [inline src/apps/Template/Template.jsx]: separate component?

File: src/common/Editor/Editor.jsx [hunk 4]
```diff
@@ -1,5 +1,3 @@
-// Port from react-ace
```
Comment (patrykkopycinski) [inline src/common/Editor/Editor.jsx]: please use `react-ace` instead

File: src/common/Notification/Notification.jsx [hunk 5]
```diff
@@ -76,6 +76,24 @@ const Notification = React.createClass({
     this.setState({ isNotificationVisible: false });
   },
 
+  renderCloseButton() {
```
Comment (patrykkopycinski) [inline src/common/Notification/Notification.jsx]: separate component?

File: src/apps/Template/Template.jsx (hunk 3 above)
Comment (wpasinski) [inline src/apps/Template/Template.jsx]: are you sure it's making any difference? `<TemplateCodeEditor>` will contain `<Editor>` with all params the same. I moved it to a separate method just to make main `render()` more readable.

**Comment by patrykkopycinski** [comment]
@wpasinski what about that?

File: src/common/Editor/Editor.jsx (hunk 4 above)
Comment (wpasinski) [inline src/common/Editor/Editor.jsx]: I suggest to have another ticket for that. It won't be a simple replace. Also whole saving mixin will need a update.

**Comment by wpasinski** [comment]
@gh3r tests fixed.

**Pull request merged**